packages/core/components/NestedDropdown/nesteddropdown.styles.css

```diff
@@ -0,0 +1,81 @@
+.nested-dropdown {
+  li {
+    list-style: none;
+    cursor: pointer;
+
+    &:focus {
+      border: 2px solid orange;
+    }
+  }
+
+  .searchInput {
+    border: none;
+    position: relative;
+    display: inline-block;
+    width: 100%;
+  }
+
+  .nested-dropdown__option {
+    padding: 7px 15px;
+    background: white;
+    border: 1px solid #ccc;
```

Comment:
we have global css variables that should be used. i think this would be var(--lightGray)? https://github.com/CDCgov/cdc-open-viz/blob/edde49c96dee146de5e3a4537880b1bcf4dbee08/packages/core/styles/_global-variables.scss#L5

I think you may need to rebase this branch to get those variables though, it was a recent addition